platform-core-module/src/main/java/com/softicar/platform/core/module/program/IProgram.java

```diff
@@ -18,13 +18,27 @@ default IDisplayString toDisplay() {
 	/**
 	 * This method can be overridden to return an optional cron expression.
 	 * <p>
-	 * If defined, the cron expression will be used to schedule this {@link IProgram}
-	 * automatically by {@link ProgramStandardConfiguration}.
+	 * If defined, the cron expression will be used to schedule this
+	 * {@link IProgram} automatically by {@link ProgramStandardConfiguration}.
 	 *
 	 * @return an optional cron expression
 	 */
 	default Optional<String> getDefaultCronExpression() {
 
 		return Optional.empty();
 	}
+
+	/**
+	 * This method can be overridden to return an optional description of this
+	 * {@link IProgram}.
+	 * <p>
+	 * If defined, the description will be displayed in the table of
+	 * {@link ProgramPage}.
+	 *
+	 * @return an optional description
+	 */
+	default Optional<IDisplayString> getDescription() {
```

Comment:
Fine for me, but did you consider to drop the `Optional` and just return IDisplayString.EMPTY by default? Just asking, no need to change.

Reply:
I though about that.
I think it should be similar to method `getDefaultCronExpression()`.
And for me `Optional` signals better than `IDisplayString` that `return null` is forbidden.

Reply:
ok, just wanted to talk about it